polkadot/node/subsystem-util/src/vstaging.rs

```diff
@@ -31,7 +31,7 @@ const LOG_TARGET: &'static str = "parachain::subsystem-util-vstaging";
 
 /// A snapshot of the runtime claim queue at an arbitrary relay chain block.
 #[derive(Default)]
-pub struct ClaimQueueSnapshot(BTreeMap<CoreIndex, VecDeque<ParaId>>);
+pub struct ClaimQueueSnapshot(pub BTreeMap<CoreIndex, VecDeque<ParaId>>);
```

Comment:
there aren't any invariants kept. and you suggested using the vecdeque directly: https://github.com/paritytech/polkadot-sdk/pull/4724#discussion_r1638325129
instead of adding more methods that look similar to each other I decided there's no reason why we can't expose its inner type.

Reply:
Nit: Alternatively you could use: derive_more::Deref for read only access